Rock/Web/UI/Controls/Resource Controls/ResourceAreaRow.cs

```diff
@@ -279,7 +279,11 @@ public override void RenderControl( HtmlTextWriter writer )
 
             _lbAddArea?.RenderControl( writer );
             writer.Write( " " );
-            _lbAddGroup?.RenderControl( writer );
```

Comment:
Our plan was to start using c# 6.0 features in the .dlls now. We can't use it in the web project yet as we'd need to wait until the hosting companies all support it.